### DISCUSSION TITLE
Error in MEEP simulation

### DISCUSSION THREAD

**Discussion by mamiyaakira** [NanoComp/meep#2228] [Q&A]
sim.run (until _ after _ sources = 100)

Now that we've done the above,
An error similar to the following was output:.
meep: error on line 305 of h5 file.cpp: error opening HDF5 input file

When does this error occur?
The MEEP program is run using openmpi.

**Comment by smartalecH** [comment]
This isn't enough information to help you debug your issue. At the very least you need to include a self-contained, minimum working example we can run ourselves that throws this error.

**Comment by oskooi** [comment]
@mamiyaakira, it seems you are new to Meep which means you should spend time carefully reviewing the user manual to learn about all the various features and how to use them. The Issues and Discussion sections of the Github repository are mainly used to report bugs and propose feature requests. This is a public forum and not a personal hotline for technical support.

**Reply by mamiyaakira** [reply]
Thank you for your reply.
All right. I will review the manual.
